pyproject.toml

```diff
@@ -96,26 +96,27 @@ filterwarnings = ["error"]
 
 [tool.coverage.run]
 omit = [
-    "venv/*", 
-    "httpcore/_sync/*"
+    "venv/*",
+    "httpcore/_sync/*",
+]
+include = [
+    "httpcore/*",
+    "tests/*",
 ]
-include = ["httpcore/*", "tests/*"]
 
 [tool.ruff]
 exclude = [
     "httpcore/_sync",
     "tests/_sync",
 ]
```

Comment:
Why these folders excluded?

Reply:
https://github.com/encode/httpcore/pull/887/commits/2e95f26a2b787063549f5ad6042099cf1ac7b3de

Reply:
Because they're auto-generated... they mirror the async codebase.